File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By 

with  webdriver.Firefox() as driver :
    driver.get("https://training-support.net")
    print(f"Page title is : {driver.title}")
    element = driver.find_element(By.LINK_TEXT,"About Us")
    element.click()
    print("About Us has been clicked")
    print(f"New Page title is : {driver.title}")
    driver.quit()

File: Selenium/Activities/Activity1.java
package examples;

import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class Activity1 {
	
	public static void main(String[] args) {
		
		WebDriver driver = new FirefoxDriver();
		driver.get("https://training-support.net");
		System.out.println("Page title is : "+driver.getTitle());
		WebElement element = driver.findElement(By.linkText("About Us"));
		element.click();
		System.out.println("About Us has been clicked");
		System.out.println("New Page title is : "+driver.getTitle());
	    driver.quit();
	}

}
